backend/coins/btc/synchronizer/synchronizer.go

```diff
@@ -76,3 +76,9 @@ func (synchronizer *Synchronizer) WaitSynchronized() {
 	}
 	<-wait
 }
+
+// TODO
+func (synchronizer *Synchronizer) IsSynced() bool {
+	defer synchronizer.waitLock.RLock()()
+	return synchronizer.requestsCounter == 0
```

Comment:
Fyi, this is not the same as account.Synced(), as account.Synced() flips to true the first time the request counter is 0 and stays true afterwards.

I think we could probably just remove this check, as account.go already checks it before calling any of the functions below, right?

Reply:
>  which calls "handlers.account.Balance()"  [...]  so it never cheks for "account.Synced()"

But it does, you added the check yourself to account.Balance() in this PR :smile: You return `ErrSyncInProgress` in that case.

Reply:
Yeah I was misguided by the code flow here and was looking at the wrong place :)